pkg/raft/testdata/heartbeat_resp_recovers_from_probing.txt

```diff
@@ -47,28 +47,32 @@ tick-heartbeat 1
 ----
 ok
 
-# Heartbeat -> HeartbeatResp -> MsgApp -> MsgAppResp -> StateReplicate.
+# Heartbeat -> MsgApp -> MsgAppResp -> StateReplicate.
 stabilize
 ----
 > 1 handling Ready
   Ready MustSync=false:
   Messages:
   1->2 MsgHeartbeat Term:1 Log:0/0
   1->3 MsgHeartbeat Term:1 Log:0/0
+  1->2 MsgApp Term:1 Log:1/11 Commit:11
 > 2 receiving messages
   1->2 MsgHeartbeat Term:1 Log:0/0
+  1->2 MsgApp Term:1 Log:1/11 Commit:11
 > 3 receiving messages
   1->3 MsgHeartbeat Term:1 Log:0/0
 > 2 handling Ready
   Ready MustSync=false:
   Messages:
   2->1 MsgHeartbeatResp Term:1 Log:0/0
+  2->1 MsgAppResp Term:1 Log:0/11 Commit:11
 > 3 handling Ready
   Ready MustSync=false:
   Messages:
   3->1 MsgHeartbeatResp Term:1 Log:0/0
 > 1 receiving messages
   2->1 MsgHeartbeatResp Term:1 Log:0/0
+  2->1 MsgAppResp Term:1 Log:0/11 Commit:11
   3->1 MsgHeartbeatResp Term:1 Log:0/0
 > 1 handling Ready
```

Comment:
Why are we sending another `MsgApp` below? There hasn't been a tick, and everything seems in flight. Is the the `MsgHeartbeatResp` logic mixing with the liveness-based one?